internal/conformance/runconformance.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
```

Comment:
Any objections to the full `set -euo pipefail`? Erroring on undefined variables seems nice, and some extra safety around pipes will be nice if we ever add one later on.

Shell scripts aren't my favorite, so I prefer maximum strictness. If you haven't already, please also run this through `shellcheck`.

Reply:
Sure, that's fine. I am fully aware of that "strict mode" setting, but am not in a habit of using it because I often use `grep` in scripts, which is a serious pain to use with `set -o pipefail` since it exits non-zero if there are no matches... (Writing `(grep ... || true) | xargs ..` is pretty gross.)

Done.